JH on opf (#82)
aicoe-aiops/data-science-workflows: docs/JH_access.md

# Deploy Jupyter Lab or Notebooks easily with Operate First

In this guide, we will go over how you can spin up a classic Jupyter Notebook or a Jupyterlab instance to get started with your very own cloud based data science development environment.

> * To get started, first go to the [operate-first](https://www.operate-first.cloud/) website.

Reviewer: ```suggestion
* To get started, first go to the [operate-first](https://www.operate-first.cloud/) website and navigate to `Community Handbook` > `GitOps Docs` > `Documentation`. Here you can find info about all the clusters and services managed by the OPF team. 
```

> * To find the resources to access Jupyterhub and Jupyterlab instances, go  to [Open Data Hub](https://www.operate-first.cloud/apps/content/odh/README.html).

Reviewer: ```suggestion
* To find the resources to access JupyterHub, select `Open Data Hub (ODH)` from the menu on the left  to [Open Data Hub](https://www.operate-first.cloud/apps/content/odh/README.html). Then click on the [UI link]( https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud). 
```

> * Here, you can find all the applications that are enabled on the cluster. Let’s start by clicking on the link for the [JupyterHub application](https://jupyterhub-opf-jupyterhub.apps.smaug.na.operate-first.cloud/).

Reviewer: ```suggestion

```

    ![Environment Variables](./../public/assets/Setup_Environment/env_variables.png)

* Once you’ve selected the options that are right for you, go ahead and click “Start server”.

* Starting the instance may take a few seconds, up to a minute. While it’s loading, you can examine the event logs to see what’s going on. One thing to note here is that the jupyter server is connecting to a storage volume, which is like your local hard drive, where your notebooks and other files are stored. So as you log on and off or shut down your server completely, your work persists, even if you start a different notebook image later on - it’s really like your own personal drive.

    ![Event logs](./../public/assets/Setup_Environment/event_logs.png)

* When you launch your jupyterlab environment for the first time it will be empty. However, the instance here has connected to the storage volume associated with an existing account and so shows some projects that the user has been working on. This page provides a nice user-friendly UI to navigate the notebooks or other files you have in the remote storage. 

    ![JH start page](./../public/assets/Setup_Environment/jh_start_page.png)

> * Now assuming your directory is completely empty, let’s go ahead and create a new notebook. We can click “+” in the upper left-hand corner, then select python3 from the notebook section. This will spawn a new empty “untitled” notebook for us.

Reviewer: Let's update this instruction for jupyter lab.

Author: Added :+1:

* Now you can start developing. With notebooks, you can toggle cells as either markdown for annotating our work,  display plots, graphs, or images, and write and run python code as we normally would. You’ve got your very own cloud-based data science working environment!  

* You can rename and save the notebook and then shut it down. Once the notebook is closed, you can see that it’s still running on the server based on this green icon next to the file name. 

    ![Running green icon](./../public/assets/Setup_Environment/green_icon.png)

    ![Running Tab](./../public/assets/Setup_Environment/running_tab.png)

* Going to the running tab, we can shut it down completely. 

> * This was the classic notebook environment that you’re probably used to.  We can also use a jupyter lab environment with its enhanced UI and a larger selection of extensions. Jupyter lab is great, it retains all the benefits of the classic notebooks but with a little more flexible and feature-rich UI as well as a whole host of additional extensions to make life easier for data science development.

Reviewer: ```suggestion
*You can also use a classic jupyter notebook environment, however, we  typically prefer the jupyter lab environment with its enhanced UI and a larger selection of extensions. Jupyter lab retains all the benefits of the classic notebooks but with a little more flexible and feature-rich UI as well as a whole host of additional extensions to make life easier for data science development.
```
